opm/simulators/linalg/PreconditionerFactory.hpp

```diff
@@ -156,26 +156,20 @@ class PreconditionerFactory
         using C = Comm;
         doAddCreator("ILU0", [](const O& op, const P& prm, const C& comm) {
             const double w = prm.get<double>("relaxation");
-#if DUNE_VERSION_NEWER(DUNE_ISTL, 2, 7)
-            return wrapBlockPreconditioner<DummyUpdatePreconditioner<SeqILU<M, V, V>>>(comm, op.getmat(), w);
-#else
-            return wrapBlockPreconditioner<DummyUpdatePreconditioner<SeqILU0<M, V, V>>>(comm, op.getmat(), w);
-#endif
+            return std::make_shared<Opm::ParallelOverlappingILU0<M, V, V, C>>(
```

Comment:
This change (together with the "ILUn" change below) seems to eliminate the option to run the Block Jacobi/SeqILU preconditioner combination?

Reply:
Well, flow does not use that combination either (see ISTLSolverEbos.hpp). If I am not completely mistaking `ParallelOverlappingILU0` should do the same if the matrix is correctly set up. My aim is to bring the codes paths closer together as they have been different  from the beginning and are diverging more and more...

The previous version using `DummyUpdatePreconditioner` never recalculated the decomposition as its update method is a no-op.

Reply:
> Well, flow does not use that combination either (see ISTLSolverEbos.hpp). If I am not completely mistaking `ParallelOverlappingILU0` should do the same if the matrix is correctly set up. My aim is to bring the codes paths closer together as they have been different from the beginning and are diverging more and more...

I know regular Flow does not use that combination, just thought it was nice to offer the option (more options to experiment with is better). Which code paths is it you want to bring together?